### PR TITLE
Make wrapito agnostic of testing framework

### PR DESCRIPTION
**Pull request by arturosdg**
## :tophat: What?

Mirror of this PR: https://github.com/mercadona/wrapito-vitest/pull/56

ADD WARNING ABOUT THE BREAKING CHANGE OF MULTIPLE HOST WITH API VERSION

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -1,23 +1,30 @@
 # This workflow will run tests using node and then publish a package to NPM when a tag is pushed
 
-name: Publish wrapito to npm
+name: Publish wrapito-vitest to npm
```
Comment (orestesrequena) [inline .github/workflows/npm-publish.yml:3]: is this just wrapito ?

File: tests/lib/debugRequests.test.ts [hunk 2]
```diff
@@ -69,13 +57,13 @@ it('should warn about the code making a request that has not being mocked', asyn
...
 it('should warn about the code making a request that has not being mocked enough times', async () => {
-  const consoleWarn = jest.spyOn(console, 'warn').mockImplementation()
+  const consoleWarn = vi.spyOn(console, 'warn')
   configure({ mount: render })
   wrap(MyComponentWithFeedback)
     .withNetwork({
       host: 'my-host',
       path: '/path/to/save/',
-      method: 'post',
+      method: 'POST',
```
Comment (HannaSyn) [inline tests/lib/debugRequests.test.ts:66]: is it case sensitive?

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-v18.17.1
+nodejs 18.18.0
```
[the remark this answers is not in the document]
Comment (arturosdg) [reply .nvmrc:1]: Nope, fixing it

File: .github/workflows/npm-publish.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (arturosdg) [reply .github/workflows/npm-publish.yml:3]: Yep, thanks!

File: tests/lib/debugRequests.test.ts (hunk 2 above)
Comment (arturosdg) [reply tests/lib/debugRequests.test.ts:66]: Not anymore, lowercase should work too, changing it

**Pull request merged**